src/strategies/dogs-unchained/index.ts

```diff
@@ -108,46 +107,61 @@ export async function strategy(
     totalBOOM += parseFloat(formatUnits(value.toString(), 18));
   });
 
+  const convertAddress = (address) => {
+    for (const i in addresses) {
+      if (addresses[i].toLowerCase() == address.toLowerCase())
+        return addresses[i];
+    }
+    return address;
+  };
+
```

Comment:
Hey @dogsunchained How about Checksum all addresses everywhere using https://docs.ethers.io/v5/api/utils/address/#utils-getAddress
Like here 
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/gno/index.ts#L37

Reply:
just made the edit!